torch/csrc/autograd/variable.cpp

```diff
@@ -452,6 +457,12 @@ void handle_view_on_rebase(DifferentiableViewMeta* diff_view_meta, bool indirect
                        " autograd logic to handle view+inplace would override the custom backward associated with the custom"
                        " Function, leading to incorrect gradients. This behavior is deprecated and will be forbidden starting"
                        " version 1.6. You can remove this warning by cloning the output of the custom Function.");
+      } else if (diff_view_meta->creation_meta == CreationMeta::MULTI_OUTPUT_SAFE) {
+        msg = c10::str(msg, " This view is the output of a function that "
+                       "returns multiple views. Inplace operators on such "
+                       "views are being deprecated and soon will not be "
```

Comment:
I would be more aggressive here and say `v1.8` (given that this will go into v1.7 most likely.